src/apps/leaderboards/models.py

```diff
@@ -3,10 +3,26 @@
 
 
 class Leaderboard(models.Model):
+    ADD = "Add"
+    ADD_DELETE = "Add_And_Delete"
+    ADD_BY_REVIEW = "Add_By_Review"
+    FORCE_LAST = "Force_Last"
+    FORCE_LATEST_MULTIPLE = "Force_Latest_Multiple"
+    FORCE_BEST = "Force_Best"
+    SUBMISSION_RULES = (
+        (ADD, "Only allow adding one submission"),
+        (ADD_DELETE, "Allow users to add a single submission and remove that submission"),
+        (ADD_BY_REVIEW, "Only allow adding one submission that is approved by review"),
+        (FORCE_LAST, "Force only the last submission"),
+        (FORCE_LATEST_MULTIPLE, "Force latest submission to be added to leaderboard (multiple)"),
+        (FORCE_LATEST_MULTIPLE, 'Force only the best submission to the leaderboard')
+    )
+
```

Comment:
Is there a plan to impose the remainder of these in the future?

Reply:
Yes, #513 and #511

Reply:
👌🏼